app/frontend/src/components/MapSearch.tsx

```diff
@@ -135,28 +154,33 @@ export default function MapSearch({
         className={classes.form}
       >
         <Autocomplete
-          label="My location"
+          label="Search for location"
           value={value}
           size="small"
           options={searchOptions.map((o) => o.name)}
           loading={searchOptionsLoading}
           open={open}
           onBlur={() => setOpen(false)}
           onChange={(e, inputValue, reason) => {
+            // setAddress(inputValue ?? "", inputValue ?? "");
             setValue(inputValue ?? "");
             searchSubmit(inputValue ?? "", reason);
           }}
-          onInputChange={(_, inputValue) => setValue(inputValue)}
+          // onInputChange={(_, inputValue) => setAddress(inputValue, inputValue)}
           freeSolo
           multiple={false}
+          // show all returned results, don't do a filter client side
+          filterOptions={(x) => x}
```

Comment:
can't this just be omitted? Or is there a default filtering function?

Reply:
Oh! Is that why "No results" wasn't displaying????

Reply:
> Oh! Is that why "No results" wasn't displaying????

Yep ;)